charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 3.98.2
+
+* Add the `NVIDIA_VISIBLE_DEVICES` environment variable to the containers when GPU monitoring is enabled, as it might be needed if the NVIDIA device plugin does not have `accept-nvidia-visible-devices-as-volume-mount` enabled.
```

Comment:
nit: i would write here the explicit meaning: device plugin doesn't support gpu device mounting via volume mounts and mention in the brackets the name of the parameter (`accept-nvidia-visible-devices-as-volume-mount`)